knowledge-base/disable-build-new-data-connections-wrd.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Disable "Build new data connections" in the SqlDataSource Wizard of the Web Report Designer
+description: Learn how to restrict the "Build new data connections" option in the "Choose data connection" window of the SqlDataSource wizard of the Web Report Designer while allowing users to select from existing data connections.
+type: how-to
+page_title: How to Disable "Build New Data Connections" in Web Report Designer
+meta_title: How to Disable "Build New Data Connections" in Web Report Designer
+slug: restricting-build-new-data-connections-report-designer
+tags: reporting, reportdesigner, datasource, connections, middleware
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td> Product </td>
+            <td> Reporting </td>
+        </tr>
+        <tr>
+            <td> Version </td>
+            <td> 19.2.25.1001 </td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I need to prevent users from creating new database connections in the Web Report Designer's SqlDataSource component while still allowing them to select from pre-configured existing connections.
```

Comment:
As you use "SqlDataSource" in the article's metadata, go for "SQL DataSource" in the article's body - this allows you to optimize the article for both writing styles.
```suggestion
I need to prevent users from creating new database connections in the Web Report Designer's SQL DataSource component while still allowing them to select from pre-configured existing connections.
```